scanner/src/test/kotlin/scanners/fossid/FossIdTest.kt

```diff
@@ -509,7 +553,13 @@ class FossIdTest : WordSpec({
                 service.createScan(USER, API_KEY, projectCode, scanCode, vcsInfo.url, vcsInfo.revision)
                 service.downloadFromGit(USER, API_KEY, scanCode)
                 service.checkDownloadStatus(USER, API_KEY, scanCode)
-                service.runScan(USER, API_KEY, scanCode, *FossId.deltaScanRunParameters(originCode))
+                service.runScan(
+                    USER, API_KEY, scanCode, *FossId.deltaScanRunParameters(originCode),
```

Comment:
I'm surprised that *detekt* doesn't complain here to put these arguments also into separate lines, like below. Would you mind still changing that?